src/main/groovy/com/google/appengine/AppEnginePlugin.groovy

```diff
@@ -245,28 +256,32 @@ class AppEnginePlugin implements Plugin<Project> {
         }
     }
 
-    private void configureAppConfig(Project project, AppEnginePluginExtension appEnginePluginExtension) {
+    private void configureWorkingDirectory(Project project) {
+        AppConfigTaskTemplate.metaClass.getWorkingDirectory = { -> this.explodedAppDirectory }
```

Comment:
I would prefer if "workingDirectory" was defined in WebAppDirTask instead of via a metaClass here, we don't really need to complicate the code if we know what we want to do.